src/emulator/modules/SceGxm/SceGxm.cpp

```diff
@@ -1049,16 +1060,55 @@ EXPORT(int, sceGxmSetFragmentTexture, SceGxmContext *context, unsigned int textu
         constexpr auto A_max = std::numeric_limits<decltype(A)>::max();
 
         switch (texture::get_swizzle(fmt)) {
-        case SCE_GXM_TEXTURE_SWIZZLE4_ABGR: R = 0; G = 1; B = 2; A = 3; break;
```

Comment:
As I've already told you on Discord (just leaving this here so you remember it), you need to remove this diff and if you want, add the `// clang ...` stuff. Else just leave it as is (do change it back to the compact version though) and Rin will add those 2 lines in reformatting PR.